### ISSUE TITLE
Combine Telegram notifications to DEXBot

### ISSUE THREAD

**Issue by MarkoPaasila** [Codaone/DEXBot#418]
Add [this](https://github.com/Codaone/BitShares-Telegram-notifier) functionality to DEXBot. We already have the accounts, but an interface is necessary to add the API id.

**Comment by bitfag** [comment]
What kind of notifications do you mean? Just account updates or messages from dexbot about what it is doing?

**Comment by MarkoPaasila** [comment]
It sends a message to telegram each time a trade happens. It's a user friendly way of keeping an eye on the activity.

**Comment by thehapax** [comment]
I would like to assign this task to myself.

**Comment by MarkoPaasila** [comment]
Would it be good to change the format to:
```
**Account**
+ 1.2345678 BTC
-12345.6789 BTS
```
Instead of the current:
```
**account:** sold 0.15457 BTS 0.020 STEEM @ 0.12939121 STEEM/BTS
```
It's hard to decide which way to report price.

**Comment by MarkoPaasila** [comment]
If this is done in relation to a worker, then we would know which way to report price, in which case the format could be:
```
**worker**
+ 1.23456789 BTC
- 12345.6789 BTS
**price:** 10000 BTS/BTC
```

**Comment by bitfag** [comment]
Always report price in BASE, e.g. xxxx BASE/QUOTE (0.00010000 BTC/BTS from your example)?

**Comment by MarkoPaasila** [comment]
Yes I think that would be good, as the user supposedly decides which asset is his unit of measure (base) when configuring the strategy.

**Comment by thehapax** [comment]
@PermieBTS is Gabriel now working on this? or separate project? Happy to reassign task as my plate is too full

**Comment by MarkoPaasila** [comment]
Could it be a good idea to avoid pulling in lots of third party libraries if possible? After all, the bot holds the keys to quite some value.

Could telegram notifications make use of the eventual api?

**Comment by PermieBTS** [comment]
Yes Gabriel tells me he is working on this functionality
